projects/driver_controls/inc/can_output.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// Module for broadcasting CAN messages
+#include <stdint.h>
+#include "status.h"
+#include "event_arbiter.h"
+
+// Message ID definitions for the input devices
+typedef enum {
+  CAN_OUTPUT_MESSAGE_POWER = 0,
+  CAN_OUTPUT_MESSAGE_PEDAL,
+  CAN_OUTPUT_MESSAGE_DIRECTION_SELECTOR,
+  CAN_OUTPUT_MESSAGE_TURN_SIGNAL,
+  CAN_OUTPUT_MESSAGE_HAZARD_LIGHT,
+  CAN_OUTPUT_MESSAGE_MECHANICAL_BRAKE,
+  CAN_OUTPUT_MESSAGE_HORN,
+  CAN_OUTPUT_MESSAGE_PUSH_TO_TALK,
+  NUM_CAN_OUTPUT_MESSAGES
+} CANOutputMessage;
```

Comment:
As discussed earlier today please update:
IIRC 
- All lights can be one message.
- Power state message is needed for Chaos.
- I didn't think Push to Talk was on CAN? Isn't the connector for this on the steering wheel which is connected to this board?

Reply:
Yeah, sorry. Those were some changes that I forgot to make for driver controls. I can just have the light-related inputs broadcast under one lights message. Also, power state messages were going to be sent under `CAN_OUTPUT_MESSAGE_POWER`.

Reply:
PTT could be for debugging. Mechanical brake is also for debugging?

For lights, if you're joining them in the same message, I'd recommend ensuring that your different sources don't collide. I don't think there's anything checking if the hazard lights and turn signals are different IDs.

I assume the pedal message is for the motor controllers? We could probably join them with the direction selector since we wouldn't want to go the wrong direction by accident.

Reply:
@Derpalooza - Please check that your lights messages aren't colliding.

Reply:
Actually, rather than having `CANOutputMessage` reflect the ID of the target system, I think it might be better to use the enums as an identifier for the device that initiated the CAN transfer.
We can use the `CANOutputMessage` value that was passed in to determine the id of the target system, which would allow us to build a CAN message, as well as distinguish which input device was responsible for the event.

Reply:
Take a look at our codegen - the `CAN_PACK` or `CAN_TRANSMIT` functions might be useful.

Reply:
Not sure if these comments are still valid? Are you blocked on codegen being integrated into our builds?

Reply:
I'm fine with merging these first, as long as there is a `TODO` to do the work once codegen is integrated.